lib/ansible/plugins/callback/nrdp.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# (c) 2018 Remi Verchere <remi@verchere.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: nrdp
+    type: notification
+    author: "Remi VERCHERE (@rverchere)"
+    short_description: post task result to a nagios server through nrdp
+    description:
+        - this callback send playbook result to nagios
+        - nagios shall use NRDP to recive passive events
+        - the passive check is sent to a dedicated host/service for ansible
+    version_added: 2.7
+    requirements:
+        - requests
+    options:
+        url:
+            description: url of the nrdp server
+            required: True
+            env:
+                - name : NRDP_URL
+        token:
+            description: token to be allowed to push nrdp events
+            required: True
+            env:
+                - name: NRDP_TOKEN
+        hostname:
+            description: hostname where the passive check is linked to
+            required: True
+            env:
+                - name : NRDP_ANSIBLE_HOSTNAME
+        servicename:
+            description: service where the passive check is linked to
+            required: True
+            env:
+                - name : NRDP_ANSIBLE_SERVICENAME
```

Comment:
Would be a nice to have if the plugin allowed an `ini` file configuration instead of only env vars.

Reply:
Where should I put the ini file?